modules/ROOT/pages/migration-from-5x.adoc

```diff
@@ -3,88 +3,614 @@
++
+`plugins: 'advlist list image help wordcount'`
++
+. A comma-separated string:
++
+`plugins: 'advlist,list,image,help,wordcount'`
 
-* xref:cloud-delivery[Cloud Delivery].
-* xref:option-setting-changes[Option/setting changes].
-* xref:plugin-changes[Plugin Changes]
+Previously, two other formats were allowed: comma-separated strings with spaces and arrays of space-separated strings.
 
-[[cloud-delivery]]
-=== Cloud Delivery
+Neither of these formats are allowed in {productname} 6.0.
 
-To use TinyMCE 6 from the {cloudname}, include this script in your HTML page. Replace 'no-api-key' with link:{accountsignup}/[your API key].
+=== Removed plugins
 
-[source,html,subs="attributes+"]
+All references to any of these plugins should be removed from plugin configurations.
+
+NOTE: Leaving references to removed plugins in a plugin configuration will cause console errors documenting the plugin failing to load. {productname} 6.0 should, otherwise, work without error. If, however, {productname} 6.0 is ‘installed’ incorrectly — by being unZipped onto an existing {productname} 5.0 instance for example — the old plugin will not be removed, it will load and, since a {productname} 5.x plugin is attempting to work with a {productname} 6.0 instance, error notifications will almost certainly present to the end-user.
+
+See below for specific documentation of potential migration issues, if any, regarding these removed plugins.
+
+Seven `plugins` were removed because their functionality is now a Core part of {productname} 6.0:
+
+. `colorpicker`
+. `contextmenu`
+. `hr`
+. `noneditable`
+. `paste`
+. `print`
+. `textcolor`
+
+Three plugins were removed as per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[{productname} 5 announcement, window="_blank"]:
+
+. `fullpage`
+. `bbcode`
+. `legacyoutput`
+
+These plugins were already empty. As of {productname} 6.0, however, they are gone entirely.
+
+One plugin was removed as no longer supported:
+
+. `spellchecker`
+
+NOTE: The https://tiny.cloud/tinymce/features/#productivity[Premium plugin], https://tiny.cloud/docs/plugins/premium/tinymcespellchecker/[Spellchecker Pro], which offers equivalent functionality and more, is available.
+
+NOTE: No Premium Plugins released with {productname} 6.0 are backwards compatible with earlier versions of {productname}.
+
+=== ImageTools
+
+As noted in the xref:6.0-release-notes.adoc#removed-or-deprecated-plugins-imagetools[_{productname} 6.0 Release Notes_], the `imagetools` plugin is no longer included as part of the Free editor.
+
+_ImageTools_ is now _EditImage_, a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].
+
+Also, the `editimage_proxy` option has been renamed to `export_image_proxy_service_url`. The option was renamed to make it easier to configure when using https://tiny.cloud/docs/enterprise/server/[Tiny services].
+
+Finally, `editimage` also offers a new export plugin option: `export_image_proxy_service_url`.
```

Comment:
This isn't correct either as noted above. The `export_image_proxy_service_url` is a new option for the `export` plugin.